lib/rubocop/cop/rspec/focus.rb

```diff
@@ -87,6 +87,10 @@ def correct_send(corrector, focus)
           corrector.replace(range,
                             range.source.sub(focus.method_name.to_s, unfocused))
         end
+
+        def should_autocorrect
```

Comment:
I can't find where this is called from. Either in `rubocop-rspec` or in `rubocop`.
Will specs fail if you remove this method?